tests/tests/swfs/avm2/sub_super_same_field/test.toml

```diff
@@ -1,3 +1 @@
 num_frames = 1
-# Ruffle does not currently reproduce this behavior
-known_failure = true
```

Comment:
just to be sure, I'm assuming that this is (the first) test that depends on the optimizer?